tree/ntuple/v7/inc/ROOT/RNTupleUtil.hxx

```diff
@@ -189,7 +189,7 @@ public:
    };
 
 private:
-   std::uint64_t fBytesOnStorage = 0;
+   std::uint64_t fNBytesOnDisk = 0;
```

Comment:
This gives up a bit of abstraction, for example when storing on object stores...